src/content/roadmap/user-experience/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Improving user experience
+description: It is still too complex to use Ethereum for most people. To encourage mass adoption, Ethereum must drastically lower its barriers to entry - users must get the benefits of decentralized, permissionless and censorship resistant access to Ethereum but it must be as frictionless as using a traditional web2 app.
+lang: en
+image: ../../../assets/enterprise-eth.png
+alt: "Ethereum roadmap"
+template: roadmap
+---
+
+It is still too complex to use Ethereum for most people - key management, wallets, transactions etc are too technical compared to using familiar centralized services such as online banking. The full benefits of running an Ethereum node remain out of reach for the majority of users because of the technical and hardware requirements and it requires some degree of motivation to engage with Ethereum and learn the foundational principles. This situation is fine for technically-minded Ethereum enthusiasts but it will prevent Ethereum being used day-to-day by the general public. To encourage mass adoption, Ethereum must drastically lower its barriers to entry - users must get the benefits of decentralized, permissionless and censorship resistant access to Ethereum but it must be as frictionless as using traditional web2 apps.
+
+Read on to discover some of the changes that will be implemented to allow the barriers-to-entry to be broken down:
+
+## Beyond seed phrases {#no-more-seed-phrases}
+
+Ethereum accounts are protected by a pair of keys that are used to identify accounts (public key) and sign messages (private key). A private key works like a master password that allows complete access to an Ethereum account - allowing funds to be moved. This is not a normal mode of operation for people more familiar with banks and web2 companies who manage accounts on a users behalf after retrieving and storing their personal data. In order for Ethereum to reach mass adoption without relying on centralized third-parties, there must be a very easy, frictionless way for a user to custody their own assets and keep control of their own data, but without having to understand public-private key cryptography and key management.
+
+The solution to this is using smart contract wallets to interact with Ethereum. This creates powerful new opportunities for fraud detection and defense, upgradeability and ways for accounts to be protected if the keys are lost or stolen. Although smart contract wallets exist today, they are awkward to build because they are not very well supported by the Ethereum protocol. Adding the necessary support will lead to great security and useability benefits for ordinary users. This additional support is known as **account abstraction**.
```

Comment:
```suggestion
The solution to this is using smart contract wallets to interact with Ethereum. Smart contract wallets create ways to protect accounts if the keys are lost or stolen, opportunities for better fraud detection and defense, and allow wallets to get new functionality. Although smart contract wallets exist today, they are awkward to build because the Ethereum protocol needs to support them better. This additional support is known as account abstraction.
```